### DISCUSSION TITLE
Next.js Image Optimization Increases Filesize Instead of Decreasing it

### DISCUSSION THREAD

**Discussion by designerdarpan** [vercel/next.js#29722] [Help]
I have fetched uploaded images from an api that originally has file size of 208kb with resolution of 3488*2633. I have used below code to fetch image from external api

```
const ExternalImage = forwardRef((props,ref) => {
const {src,quality,folder,...rest} = props;
const [source,setSource] = useState(`${process.env.IMAGE_DOMAIN}/${folder}/${src}`)
return (
    <Image
        src={source}
        quality={quality != undefined ? quality : 100}
        {...rest}
        onError={() => setSource(NotFound)}
    />
)
})
```

Now when I see the network tab, the same image that browser is fetching is 1 Mb. I tried to cross check if it is true, I downloaded the image that next js optimizes and found that the image file size has increased instead of decreasing.

How is this even optimizing? Am I missing anything in code?

My next.js config:

```
module.exports = {
env: {
  API_DOMAIN: 'https://api.somedomain.com/web-api/v1.0',
  IMAGE_DOMAIN: 'https://api.somedomain.com',
},
images: {
  domains: ['https://api.somedomain.com/','api.somedomain.com'],
  deviceSizes: [640, 750, 828, 1080, 1200, 1920, 2048, 3840],
  imageSizes: [16, 32, 48, 64, 96, 128, 256, 384],
},
}
```

**Comment by wwwdepot** [comment]
quality=75 is the optimal setting for [squoosh](https://squoosh.app/), try the quality slider on their website. Setting quality=100 isn't what you think it is.
